chcemvediet/apps/inforequests/forms/obligee_action.py

```diff
@@ -355,28 +401,30 @@ def post_transition(self):
         res = super(IsItAppealDecision, self).post_transition()
 
         if not self.is_valid():
+            #todo: check this
```

Comment:
@martinmacko47 Ak je formular ne-validny tak nastane rovnaky vysledok ako ked kontrolujeme `cleaned_data`. Je to OK? Nemalo by tam byt:
```
if not self.is_valid():
    res.next = CanAddDisclosureRefusalAdvancementExtension
```

Reply:
@viliambalaz Ak si spravne pamatam, tak `self.is_valid()` moze byt false iba koli tomu, ze potrebujeme prepocitat, kolko ma wizard krokov, aj ked user este nesubmitol vsetky kroky. Vtedy teda pustame vyhodnotenie krokov, aj ked este neboli submitnute, alebo obsahuju nevalidne data.

Preto vetva v `if not self.is_valid():` by mala smerovat na vetvu, ktoru v danom pripade povazujeme za najocakavanejsiu, pripadne na tu ktora je dlhsia. Pretoze podla toho aku vetvu v takom pripade zvolime, tak tolko sa uzivatelovi zobrazi, ze wizard ma krokov.

V `IsItAppealDecision` by teda "not valid" vetva mala smerovat asi na `CanAddDisclosureRefusalAdvancementExtension`, kedze tato vetva je dlhsia.